docs/content/getting-started/setup-openfga.mdx

```diff
@@ -71,6 +71,27 @@ The Playground facilitates rapid development by allowing you to visualize and mo
 
 The Playground is enabled on port 3000 by default. To disable the Playground, please take a look at [Configuring the Server](#configuring-the-server) for more information.
 
+### Profiler (pprof)
+:::caution Warning
+Continuous pprof profiling can be used in production deployments, but we recommend disabling it unless it is needed to troubleshoot specific performance or memory problems.
+:::
+
+Profiling through pprof can be enabled on the OpenFGA server by providing the `--profiler-enabled` flag.
+
+```sh
+./openfga run --profiler-enabled
```

Comment:
Since we never describe how to run / build binary in this document, should we stick with docker?  (Otherwise, we need to add in some reference on how to build/run binary)

Reply:
Good point. I copied this from the README in openfga/openfga.

I've updated it so that it uses the docker commands instead now. How does it look now?